src/less/tables.less

```diff
@@ -698,3 +698,59 @@
   }
 }
 
+@media (max-width: @global-screen-small-max) {
+
+  .ReactTable {
+    .rt-table {
```

Comment:
Hey @rosh-ttn it feels like we're not using much of the established style variables that we have and have instead targeted some very specific measurements, with lots of !important attributes that could affect CSS hierarchy in the long-term. 

I am not super familiar with the front-end design for this but I will pull the branch and provide some more concrete comments as soon as I can so we can refactor this.

Reply:
Hey @rosh-ttn I'm meeting with benwo on this today just to understand the functionality and I will run him through the progress using this branch. Hoping to have any remaining feedback to you this morning.